src/main/java/com/esri/samples/na/service_areas/README.md

```diff
@@ -0,0 +1,46 @@
+<h1>Service Area Task</h1>
+
+<p>Demonstrates how to find services areas around a point using the ServiceAreaTask. A service area shows locations that can be reached from a facility based off a certain impedance. One kind of impedence could be time.</p>

```

Comment:
"based off a certain impedance [such as travel time]". Does time impede or is it the barriers?